tests/epyccel/test_epyccel_functions.py

```diff
@@ -13,6 +13,90 @@ def clean_test():
     shutil.rmtree('__pycache__', ignore_errors=True)
     shutil.rmtree('__epyccel__', ignore_errors=True)
 
+def test_func_no_args_1(language):
+    '''test function with return value but no args'''
+    def free_gift():
+        gift = 10
+        return gift
+
+    c_gift = epyccel(free_gift, language=language)
+    assert free_gift() == c_gift()
+
+def test_func_no_args_return_type_1(language):
+    '''test return type of a function with negative return value but no args'''
+    def free_gift():
+        gift = 10
+        return gift
+
+    c_gift = epyccel(free_gift, language=language)
+    assert isinstance(free_gift(), type(c_gift()))
```

Comment:
The type and value can be tested in the same function `test_func_no_args_1` to avoid translating the function `free_gift` twice. The error message will be sufficient to tell which assertion has caused the failure